tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.Debugger.cs

```diff
@@ -146,6 +146,20 @@ internal static class Debugger
             /// </summary>
             /// <seealso cref="ExceptionReplaySettings.MaxExceptionAnalysisLimit"/>
             public const string MaxExceptionAnalysisLimit = "DD_EXCEPTION_REPLAY_MAX_EXCEPTION_ANALYSIS_LIMIT";
+
+            /// <summary>
+            /// Configuration key to enable tag code origin for span.
+            /// Default value is false.
+            /// </summary>
+            /// <seealso cref="DebuggerSettings.CodeOriginForSpansEnabled"/>
+            public const string CodeOriginForSpansEnabled = "DD_CODE_ORIGIN_FOR_SPANS_ENABLED";
+
+            /// <summary>
+            /// Configuration key for setting the number of frames to be tagged in exit span code origin.
+            /// Default value is <c>8</c>.
+            /// </summary>
+            /// <seealso cref="DebuggerSettings.CodeOriginMaxUserFrames"/>
+            public const string CodeOriginMaxUserFrames = "DD_CODE_ORIGIN_FOR_SPANS_MAX_USER_FRAMES";
```

Comment:
Given these are a debugger feature, shouldn't they have the `DD_DYNAMIC_INSTRUMENTATION_` prefix?

Also, have you added these to the allowed set in dd-go?

Reply:
1. In fact, I purposely did not put DI in the name because it can be enabled even if debugging is not enabled
2. Yes